benchmarks/infra/stage-1/README.md

```diff
@@ -1,94 +1,485 @@
-# AI on GKE Benchmark Infrastructure
```

Comment:
Looks like you've replaced the entire readme. We just update the variable section with tfdocs. Not the whole readme since it includes other instructions as well. Btw this is the command
`terraform-docs markdown table --output-file README.md --output-mode inject .`.

Reply:
This is the output of that command. It does look like the delimiter changed though? From <!-- BEGIN TFDOC --> to <!-- BEGIN_TF_DOCS --> . I removed the old section